src/main/java/org/candlepin/dto/api/v1/NestedOwnerDTOTranslator.java

```diff
@@ -53,6 +53,7 @@ public Owner populate(ModelTranslator translator, NestedOwnerDTO source, Owner d
         dest.setKey(source.getKey());
         dest.setDisplayName(source.getDisplayName());
         dest.setHref(source.getHref());
+        dest.setContentAccessMode(source.getContentAccessMode());
```

Comment:
Minor note: you don't need to add this here. This is a 'reverse' translator that translates NestedOwnerDTO coming from the API into Owner. Since your change is only about adding this field to the output (and not accepting it in the input), then let's skip this change